src/FileInput/FileInput.jsx

```diff
@@ -0,0 +1,136 @@
+import React from "react";
+import Dropzone from "react-dropzone";
+import {FlexBox, FlexItem} from "../flex";
+
+require("./FileInput.less");
+
+function DefaultIcon() {
+  return (<svg className="FileInput--Icon" width="42px" height="24px" viewBox="0 0 42 24">
```

Comment:
I kinda wonder if these SVG's ought to be defined outside this file, but I don't feel strongly about it